README.rst

```diff
@@ -110,6 +120,72 @@ Jenkins slave
           private_key: |
             -----BEGIN PGP PRIVATE KEY BLOCK-----
             ...
+Client
+------
+
+Simple client with workflow job definition
+
+.. code-block:: yaml
+
+    jenkins:
+      client:
+        master:
+          host: jenkins.example.com
+          port: 80
+          protocol: http
+        job:
+          jobname:
+            type: workflow
+            param:
+              bool_param:
+                type: boolean
+                description: true/false
+                default: true
+              string_param:
+                type: string
+                description: 1 liner
+                default: default_string
+              text_param:
+                type: text
+                description: multi-liner
+                default: default_text
+
+Groovy script samples
+
+.. code-block:: yaml
+
+    jenkins:
+      client:
+        job:
+          test_workflow_jenkins_simple:
+            type: workflow
+            display_name: Test jenkins simple workflow
+            script: |
+              node {
+                 stage 'Stage 1'
+                 echo 'Hello World 1'
+                 stage 'Stage 2'
+                 echo 'Hello World 2'
+              }
+          test_workflow_jenkins_input:
+            type: workflow
+            display_name: Test jenkins workflow inputs
+            script: |
```

Comment:
There should be support for deploying workflow job which will use SCM repository to get scripts. This is good for smaller ones but you cannot ship more complex with includable modules, etc.